<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>528</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>528</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>580</width>
    <height>528</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>System Monitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>581</width>
      <height>511</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="tabBarAutoHide">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="SysInfo">
     <attribute name="title">
      <string>SystemInfo</string>
     </attribute>
     <widget class="QLabel" name="HostName">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>450</y>
        <width>240</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>HostName:</string>
      </property>
     </widget>
     <widget class="QLabel" name="StartTime">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>440</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>StartTime:</string>
      </property>
     </widget>
     <widget class="QLabel" name="RunningTime">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>475</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>RunningTime:</string>
      </property>
     </widget>
     <widget class="QLabel" name="SysVersion">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>480</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>SystemVersion:</string>
      </property>
     </widget>
     <widget class="QLabel" name="Cpu">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>420</y>
        <width>391</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU:</string>
      </property>
     </widget>
     <widget class="QwtPlot" name="CpuPlot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>501</width>
        <height>151</height>
       </rect>
      </property>
      <property name="canvasBackground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QwtPlot" name="MemPlot">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>501</width>
        <height>151</height>
       </rect>
      </property>
      <property name="canvasBackground">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </property>
     </widget>
     <widget class="QLabel" name="CpuLabel">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CPU Usage</string>
      </property>
     </widget>
     <widget class="QLabel" name="MemLabel">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>360</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Memory Usage</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="ProcInfo">
     <attribute name="title">
      <string>Process Info</string>
     </attribute>
     <widget class="QListWidget" name="ProcList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>551</width>
        <height>411</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Ubuntu Mono</family>
        <pointsize>13</pointsize>
        <italic>false</italic>
       </font>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::DoubleClicked</set>
      </property>
      <property name="viewMode">
       <enum>QListView::IconMode</enum>
      </property>
      <property name="currentRow">
       <number>-1</number>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="ItemLabel">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>551</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>   Name                                            PID                PPID                       MemoryUse                         Priority</string>
      </property>
     </widget>
     <widget class="QPushButton" name="KillButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>480</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Kill</string>
      </property>
     </widget>
     <widget class="QPushButton" name="SearchButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="SearchText">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="RefreshButton">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>80</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="NextButton">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Next</string>
      </property>
     </widget>
     <widget class="QPushButton" name="RunButton">
      <property name="geometry">
       <rect>
        <x>369</x>
        <y>480</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ShutDownButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>480</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Shut Down System</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="State_cpuUsage">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>510</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cpu Usage</string>
    </property>
   </widget>
   <widget class="QLabel" name="State_memUsage">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>510</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Memory Usage</string>
    </property>
   </widget>
   <widget class="QLabel" name="State_currentTime">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>510</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>State_cpuUsage</zorder>
   <zorder>State_memUsage</zorder>
   <zorder>State_currentTime</zorder>
   <zorder>tabWidget</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
